### PR TITLE
Minor cleanups

### PR DESCRIPTION
**Pull request by remcohaszing**
- Remove some commented code
- Remove some redundant code
- Remove some redundant type definitions
- Remove duplicate registration of html diagnostics provider
- Avoid default exports

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -37,11 +37,6 @@ export const configureMonacoTailwindcss: typeof import('monaco-tailwindcss').con
         languageSelector,
         createHoverProvider(workerManager.getWorker),
       ),
-      registerMarkerDataProvider(
-        { editor },
```
Comment (mhsdesign) [inline src/index.ts:41]: If I see this correct - this was a duplicate?

File: src/index.ts [hunk 2]
```diff
@@ -37,11 +37,6 @@ export const configureMonacoTailwindcss: typeof import('monaco-tailwindcss').con
         languageSelector,
         createHoverProvider(workerManager.getWorker),
       ),
-      registerMarkerDataProvider(
```
Comment (mhsdesign) [inline src/index.ts:40]: If I see this correct - this was a duplicate?

Comment (mhsdesign) [inline src/index.ts:40]: If I see this correct - this was a duplicate?

File: src/types.ts [hunk 3]
```diff
@@ -8,9 +8,6 @@ import { createContext, JitContext } from 'tailwindcss/src/lib/setupContextUtils
 export interface JitState extends State {
   config: object;
   separator: string;
-  screens: string[];
-  variants: Record<string, string | null>;
-  jit: true;
```
Comment (mhsdesign) [inline src/types.ts:13]: FYI I just added this since it must be true but nvm 😇

Comment (mhsdesign) [inline src/types.ts:13]: FYI I just added this since it must be true but nvm 😇

File: src/index.ts (hunk 1 above)
Comment (remcohaszing) [reply src/index.ts:41]: Yep. I believe I added it here initially for local testing, but then added the loop below later to support other languages.

File: src/types.ts (hunk 3 above)
Comment (remcohaszing) [reply src/types.ts:13]: It could be false when it’s processed by tailwind though. I don’t know how they deal with it internall :shrug:

Comment (mhsdesign) [reply src/types.ts:13]: All fine 👍

**Comment by mhsdesign** [comment]
You don't like default exports Hu? ^^

**Pull request merged**